x/oracle/keeper/keeper.go

```diff
@@ -75,6 +74,21 @@ func (k Keeper) setProphecy(ctx sdk.Context, prophecy types.Prophecy) {
 
 // ProcessClaim ...
 func (k Keeper) ProcessClaim(ctx sdk.Context, claim types.Claim) (types.Status, error) {
+
```

Comment:
I could not find anywhere in the code where this function `ProcessClaim` gets called which leads me to believe that this will not stop non-whitelisted validators from submitting claims.

Reply:
If you run the integration tests without whitelisting a validator, you get a transaction:

```
bash-5.0# sifnodecli q tx EA6F4CCDDAC90312156CA4011A2753C0DA7FAE6754E66DC7F796A1B223AC0933
height: 11
txhash: EA6F4CCDDAC90312156CA4011A2753C0DA7FAE6754E66DC7F796A1B223AC0933
codespace: oracle
code: 10
data: ""
rawlog: 'validator must be in whitelist: failed to execute message; message index: 0'
logs: []
info: ""
gaswanted: 300000
gasused: 44836
```

Reply:
And the log has:

```
I[2020-12-21|21:43:11.366] New header 71 with hash [253 6 88 194 184 39 34 2 135 5 37 222 40 174 208 102 30 113 221 165 88 134 44 56 97 126 101 249 26 169 190 207] 
I[2020-12-21|21:43:11.373] New header 72 with hash [133 12 206 142 117 135 30 146 33 3 124 63 196 160 59 228 70 150 193 123 148 245 0 180 149 101 165 134 84 142 151 154] 
height: 0
txhash: EA6F4CCDDAC90312156CA4011A2753C0DA7FAE6754E66DC7F796A1B223AC0933
codespace: ""
code: 0
data: ""
rawlog: '[]'
logs: []
info: ""
gaswanted: 0
gasused: 0
tx: null
timestamp: ""

I[2020-12-21|21:43:14.261] Executed block                               module=state height=11 validTxs=0 invalidTxs=1
I[2020-12-21|21:43:14.267] Committed state                              module=state height=11 txs=1 appHash=FEB2E6288BACFC790CE7EF1B16911F633011C891E3E3E4D03ACD8E6BC3529C81
```

Reply:
ethbridge keeper call oracle's ProcessClaim in its ProcessClaim method.